opencensus/stats/examples/exporter_example.cc

```diff
@@ -60,10 +60,12 @@ class ExampleExporter : public opencensus::stats::StatsExporter::Handler {
         return;
       }
       std::string output;
-      absl::StrAppend(&output, "\nData for view \"", descriptor.name(),
-                      "\" from ", absl::FormatTime(view_data.start_time()),
-                      " to ", absl::FormatTime(view_data.end_time()), ":\n");
+      absl::StrAppend(&output, "\nData for view \"", descriptor.name(), "\n");
       for (const auto& row : view_data.double_data()) {
+        auto start_time = view_data.start_times().at(row.first);
+        absl::StrAppend(&output, "\nRow data \"", "\" from ",
```

Comment:
Nit: "Row data from" without the extra quotes?

Reply:
Done.